packages/ai/internal/ai.ts

```diff
@@ -0,0 +1,209 @@
+import { AI_PROMPT, Client, HUMAN_PROMPT } from "@anthropic-ai/sdk";
+import { Configuration, OpenAIApi } from "openai";
+
+let logging = true;
+
+export const setLogging = (value: boolean) => {
+  logging = value;
+};
+
+export type Message = {
+  role: "system" | "user" | "assistant";
+  content: string;
+};
+
+export type ChatOpts = {
+  instructions?: string;
+  model?: string;
+  temperature?: number;
+};
+
+export const chat = async (message: string, opts?: ChatOpts): Promise<string> => {
+  const messages: Message[] = [
+    {
+      role: "system",
+      content: getBasePrompt(opts?.instructions),
+    },
+    {
+      role: "user",
+      content: message,
+    },
+  ];
+
+  return chatInternal(messages, opts);
+};
+
+export class ChatBot {
+  history: Message[];
+  opts?: chatInternalOpts;
+
+  constructor(opts?: ChatOpts) {
+    this.history = [
+      {
+        role: "system",
+        content: getBasePrompt(opts?.instructions),
+      },
+    ];
+    this.opts = opts;
+  }
+
+  async chat(message: string): Promise<string> {
+    this.history.push({
+      role: "user",
+      content: message,
+    });
+    const response = await chatInternal(this.history, this.opts);
+    this.history.push({
+      role: "assistant",
+      content: response,
+    });
+    return response;
+  }
+}
+
+type JSONValue = string | number | boolean | JSONObject | JSONArray;
+
+interface JSONObject {
+  [x: string]: JSONValue;
+}
+
+type JSONArray = Array<JSONValue>;
+
+export class AIFunction {
```

Comment:
Let's remove the `AI` prefix since it's now an AI-focused package. I think `Function` is a reserved keyword -- if so, maybe `Func`?

```suggestion
export class Func {
```